client/components/NavBar.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react'
+import { Route, Switch, BrowserRouter, Redirect, Link } from "react-router-dom"
+
+import PetTypesList from "./PetTypesList.js"
+import AdoptablePetType from "./AdoptablePetType"
+import AdoptablePetTypeShow from './AdoptablePetTypeShow'
+import SurrenderForm from './SurrenderForm'
+
+const NavBar = props =>{
+  return(
+    <div>
+
+      <Link to='/pets'>Welcome Page</Link>
+      <Link to='/pets/cats'>Cats</Link>
+      <Link to='/pets/dogs'>Dogs</Link>
+      <Link to='/pets/foxes'>Foxes</Link>
+      <Link to="/adoption/new">Surrender Your Pet</Link>
+      
+      <Switch>
+        <Route exact path="/">
+          <Redirect to="/pets" />
+        </Route>
+        <Route exact path="/pets" component={PetTypesList} />
+        <Route exact path = "/pets/:type" component={AdoptablePetType} />
+        <Route exact path ='/pets/:type/:id' component = {AdoptablePetTypeShow} />
+        <Route exact path = '/adoptions/new' component = {SurrenderForm} />
+      </Switch>

```

Comment:
Noticing a mix of double and single quotes. I would choose one across the team to be consistent.  I'll push for double quotes.  You can install a formatter like `prettier` to help format these files on save.